packages/app/contexts/ModalContext.tsx

```diff
@@ -16,6 +16,12 @@ export enum ModalId {
   CANCEL_STACK_PROCESSING = "cancel_stack_processing",
   CANCEL_STACK_SUCCESS = "cancel_stack_success",
   SUCCESS_STACK_TOAST = "success_stack_toast",
+  APPROVE_CONFIRM = "approve_confirm",
+  APPROVE_PROCESSING = "approve_processing",
+  APPROVE_SUCCESS = "approve_success",
```

Comment:
```suggestion
  STACK_APPROVE_CONFIRM = "stack_approve_confirm",
  STACK_APPROVE_PROCESSING = "stack_approve_processing",
  STACK_APPROVE_SUCCESS = "stack_approve_success",
```

Reply:
or `APPROVE_STACK_CONFIRM`

I'm not sure what I prefer but we should be consistent on what we choose.

Reply:
fixed in [223d187](https://github.com/SwaprHQ/stackly-ui/pull/46/commits/223d187fcbdd254401795e95a9767f1a3fe8a8ec)